tools/windows-build.bat

```diff
@@ -54,7 +54,7 @@ for %%W in (wheelhouse\confluent_kafka-*cp27*win32.whl) do (
   python -c "from confluent_kafka import libversion ; print libversion()" || exit /b 1
   cd confluent-kafka-python
 
-  pytest --import-mode=append tests || exit /b 1
+  pytest -v --ignore tests/integration --timeout 20 --ignore=tmp-build --import-mode=append tests || exit /b 1
```

Comment:
is there a way to tell pytest to ignore dirs in-situ? e.g., `.pytestignore` files, pytest config,  or something?